app/models/katello/repository.rb

```diff
@@ -82,9 +80,9 @@ class Repository < Katello::Model
       :attributes => :url, :nil_allowed => proc { |o| o.custom? }, :field_name => :url,
       :if => proc { |o| o.in_default_view? }
     validates :content_type, :inclusion => {
-      :in => TYPES,
+      :in => ->(_) { Katello::RepositoryTypeManager.repository_types.keys },
       :allow_blank => false,
-      :message => (_("must be one of the following: %s") % TYPES.join(', '))
+      :message => ->(_, _) { _("must be one of the following: %s") % Katello::RepositoryTypeManager.repository_types.keys.join(', ') }
```

Comment:
If that is the typical Rails way to do this now, by all means. However, it looks to me like this is either some weird string substitution gone wrong or a confusing use of i18n.